onnx/test/test_backend_onnxruntime.py

```diff
@@ -298,6 +298,23 @@ def run_node(cls, node, inputs, device=None, outputs_info=None, **kwargs):
             "|qlinearmatmul_3D_uint8_float32"
             ")"
         )
+    if ort_version is not None and ort_version < Version("1.18"):
```

Comment:
Is there an issue to track the reason and the conditions to remove this? Maybe a `TODO:` explaining the reason